src/igvfd/schemas/mixins.json

```diff
@@ -623,5 +623,17 @@
                 "igvf_utils": "john-doe:measurement_set01, IGVFDS1222ABCD"
             }
         }
+    },
+    "cell_type_annotation": {
+        "cell_type_annotation": {
+            "title": "Cell Type Annotation",
+            "description": "The inferred cell type this file is associated with based on expression profiling.",
```

Comment:
say "single-cell" profiling because I believe you can do this with pure snATAC data as well (although it's not trivial)

Reply:
fixed